### PR TITLE
move card examples

### LINKED ISSUE #818: Example: card
Split appropriate existing examples into separate files using functional approach and icache and move into the correct `/src/examples/widgets` subdirectory.

### PR DESCRIPTION
**Pull request by vansimke**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:** Moves card examples to new format and structure

Resolves #818

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-e7ebivjv3.now.sh

File: src/examples/src/widgets/card/ActionButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const factory = create();
+
+export default factory(function ActionButtons() {
+	return (
+		<div styles={{ width: '400px' }}>
+			<Card
+				actionsRenderer={() => (
+					<div classes={cardCss.actionButtons}>
+						<Button onClick={() => console.log('action clicked.')}>Action</Button>
```
Comment (agubler) [inline src/examples/src/widgets/card/ActionButtons.tsx]: I think we are trying to render something for the user over having something logged to the console.

File: src/examples/src/widgets/card/ActionButtonsAndIcons.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Card from '@dojo/widgets/card';
+import Button from '../../../../button/index';
```
Comment (agubler) [inline src/examples/src/widgets/card/ActionButtonsAndIcons.tsx]: Same comment for these, should be using `@dojo/widgets/` and no need to specify index (and throughout the examples)

File: src/examples/src/widgets/card/CardWithMediaContent.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Card from '@dojo/widgets/card';
+import * as cardCss from '../../../../theme/card.m.css';
+
+const factory = create();
+
+export default factory(function CardWithMediaContent() {
+	return (
+		<div styles={{ width: '400px' }}>
```
Comment (agubler) [inline src/examples/src/widgets/card/CardWithMediaContent.tsx]: We can have css files for the examples so I think any example styling should be added there.

File: src/examples/src/widgets/card/ActionButtons.tsx (hunk 1 above)
Comment (vansimke) [reply src/examples/src/widgets/card/ActionButtons.tsx]: Fair enough, this was just a lift-and-shift of the old examples. I agree that updating to show a real result on the page would be better.

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-don525d65.now.sh
* Tests Deployment: https://dojowidgets-546f0v9qg.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-g0w5c5hx5.now.sh
* Tests Deployment: https://dojowidgets-of5t0kgb1.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-rcmnmr7vv.now.sh
* Tests Deployment: https://dojowidgets-of5t0kgb1.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-a7jv0q5p4.now.sh
* Tests Deployment: https://dojowidgets-mfsezh7kf.now.sh

**Pull request merged**